contracts/src/Proofs.sol

```diff
@@ -78,6 +78,72 @@ library MerkleProof {
         }
         return computedHash;
     }
+
+    /**
+     * Returns the root of a Merkle tree of all zero leaves and specified height. 
+     * A height of zero returns zero (the leaf value).
+     * A height of 1 returns the hash of two zero leaves.
+     * A height of n returns the hash of two nodes of height n-1.
+     * Height must be <= 50 (representing 2^50 leaves or 32EiB).
+     */
+    function zeroRoot(uint height) internal pure returns (bytes32) {
+        require(height <= 50, "Height must be <= 50");        
+        // These roots were generated by code in Proots.t.sol.
+        uint256[51] memory ZERO_ROOTS = [
+            0x0000000000000000000000000000000000000000000000000000000000000000,
+            0xf5a5fd42d16a20302798ef6ed309979b43003d2320d9f0e8ea9831a92759fb0b,
```

Comment:
Checks out with commp-hash-hash code I ported to go playground: https://go.dev/play/p/dICLMvpGIaA